qa/suites/rbd/nvmeof/base/install.yaml

```diff
@@ -0,0 +1,32 @@
+use_shaman: True
+tasks:
+- cephadm:
+- cephadm.shell:
+    host.a:
+    # get state before nvmeof deployment
+    - ceph orch status
+    - ceph orch ps
+    - ceph orch ls
+    - ceph orch host ls
+    - ceph orch device ls
+    - ceph osd lspools
+    # create pool
+    - ceph osd pool create mypool
+    - ceph osd pool application enable mypool rbd
+    - ceph osd lspools
+    - ceph osd pool ls detail
+    # deploy nvmeof
+    - ceph config set mgr mgr/cephadm/container_image_nvmeof quay.io/ceph/nvmeof:v0.0.5
```

Comment:
I don't think we want to hard code the tag in the test.  The latest should be pulled, otherwise it wouldn't be much of a CI ;)

Reply:
@idryomov we just use quay.io/ceph/nvmeof:latest?

Reply:
I think so!